docs/markdown.md

```diff
@@ -14,6 +14,21 @@ await $`whoami`
 await $`echo ${__dirname}`
 ```
 
+~~~js
```

Comment:
Tests should not go into docs.

Reply:
Sorry, I was mistaken because the cli.test.js run this markdown.md for checking markdown is working.
https://github.com/google/zx/blob/d6d19be1e3f77783df84c899da85d42829ef8748/test/cli.test.js#L135